src/main/java/org/terasology/minion/move/MoveAlongPathNode.java

```diff
@@ -45,6 +46,9 @@
 
     @Override
     public void construct(Actor actor) {
+        // TODO: Temporary fix for injection malfunction in actions, ideally remove this in the future.
+        pathRenderSystem = CoreRegistry.get(PathRenderSystem.class);
```

Comment:
You should remove, or at least comment out, the injection above so it's clearer.

Reply:
I commented out the `@In` above